sklearn/feature_selection/_univariate_selection.py

```diff
@@ -300,14 +295,94 @@ def f_regression(X, y, *, center=True):
     corr = safe_sparse_dot(y, X)
```

Comment:
you can remove the comment `# compute the correlation`. The code should be self explicit if the name of the variable is explicit :)